liblll/CompilerState.cpp

```diff
@@ -54,8 +54,8 @@ void CompilerState::populateStandard()
 	"(def 'msg (gaslimit to value data) { [0]:data (msg gaslimit to value 0 32) })"
 	"(def 'msg (to value data) { [0]:data (msg allgas to value 0 32) })"
 	"(def 'msg (to data) { [0]:data (msg allgas to 0 0 32) })"
-	"(def 'create (value code) { [0]:(msize) (create value @0 (lll code @0)) })"
-	"(def 'create (code) { [0]:(msize) (create 0 @0 (lll code @0)) })"
+	"(def 'create (value code) { [0]:{[0]:0xff (msize)} (create value @0 (lll code @0)) })"
+	"(def 'create (code) { [0]:{[0]:0xff (msize)} (create 0 @0 (lll code @0)) })"
```

Comment:
Is there any particular reason for using `0xff`?

I'd reorder this if possible as:
```
// NOTE: memory location 0 is set in order to force msize to be at least 32 bytes
(def 'create (code) { [0]:0 [0]:(msize) (create 0 @0 (lll code @0)) })
```